### PR TITLE
Secure URLs

### PR DESCRIPTION
**Pull request by vszakats**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/wsk.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 class Wsk < Formula
   desc "OpenWhisk Command-Line Interface (CLI)"
-  homepage "http://openwhisk.org"
+  homepage "https://openwhisk.incubator.apache.org/"
```
Comment (javian) [inline Formula/wsk.rb]: This seems to be available on https://openwhisk.apache.org/ as well - is there a preference here ?

**Review by DomT4 — APPROVED** [review]
Feel free to not bother waiting for CI on these.

**Comment by vszakats** [comment]
Thanks @DomT4 — and merged!

**Comment by vszakats** [comment]
Status update:
homebrew-core-https-ratio-2

**Comment by DomT4** [comment]
That's a pretty pretty graph 😍

**Pull request merged**